core/build.gradle.kts

```diff
@@ -397,9 +397,11 @@ tasks.withType<AbstractDokkaLeafTask>().configureEach {
     dokkaSourceSets.configureEach {
         // reportUndocumented.set(true) // much noisy output about `hashCode` and serializer encoders, decoders etc
         skipDeprecated.set(true)
+        // Enum members and undocumented toString()
+        suppressInheritedMembers.set(true)
```

Comment:
Is there a flag to also make it hide `Padding.values()` and `Padding.valueOf(String)`?

Reply:
Not to my knowledge